src/command_modules/azure-cli-rdbms/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+0.0.5 (2017-06-23)
```

Comment:
Please change this line to `unreleased`. We will change the version and date when we release packages.